test/test_pipeline/components/classification/test_passive_aggressive.py

```diff
@@ -11,16 +11,16 @@ class PassiveAggressiveComponentTest(BaseClassificationComponentTest):
     __test__ = True
 
     res = dict()
-    res["default_iris"] = 0.78
-    res["iris_n_calls"] = 4
-    res["default_iris_iterative"] = 0.98
-    res["default_iris_proba"] = 0.4471909030527266
-    res["default_iris_sparse"] = 0.36
-    res["default_digits"] = 0.9253187613843351
-    res["digits_n_calls"] = 4
+    res["default_iris"] = 0.92
```

Comment:
Do you know what caused these changed?

Reply:
Unfortunately not. Maybe the iterative training was one wrong before?